aspnetcore/blazor/fundamentals/startup.md

```diff
@@ -732,9 +732,18 @@ In `wwwroot/index.html`, remove the default SVG round indicator in `<div id="app
 
 ## Configure the .NET WebAssembly runtime
 
-To configure the .NET WebAssembly runtime, use the `configureRuntime` function with the `dotnet` runtime host builder.
+In advanced programming scenarios, the `configureRuntime` function with the `dotnet` runtime host builder is used to configure the .NET WebAssembly runtime. For example, `dotnet.withEnvironmentVariable` sets an environment variable that:
 
-The following example sets an environment variable, `CONFIGURE_RUNTIME`, to `true`:
+* Configures the .NET WebAssembly runtime.
+* Changes the behavior of a C library if the app is built with a [`NativeFileReference`](xref:blazor/webassembly-native-dependencies).
```

Comment:
this makes it sound like this is some special feature of NativeFileReference, but in reality it's just the C library reading an env var and then doing something :)

Reply:
```suggestion
* Changes the behavior of a C library.
```